docs/ref/contrib/syndication.txt

```diff
@@ -1095,3 +1112,47 @@ For example, you might start implementing an iTunes RSS feed generator like so::
 
 There's a lot more work to be done for a complete custom feed class, but the
 above example should demonstrate the basic idea.
+
+Feed stylesheets
+----------------
+
+You can add styling to your RSS feed using an `XSLT stylesheet`_ by passing
+an ``xsl_stylesheet_url`` argument to the feed's constructor.
+
+This can be a hardcoded URL::
+
+    from django.contrib.syndication.views import Feed
+
+
+    class FeedWithHardcodedStylesheet(Feed):
+        ...  # author, etc.
+        xsl_stylesheet_url = "https://example.com/rss_stylesheet.xslt"
+
+
+You can also use Django's static files system::
+
+    from django.contrib.syndication.views import Feed
+    from django.templatetags.static import static
+
+
+    class FeedWithStaticFileStylesheet(Feed):
+        ...  # author, etc.
+        xsl_stylesheet_url = static("rss_styles.xslt")
+
+
+Another option is to have a view in your project that renders the XSLT document.
+You can then link it like so::
+
+    from django.contrib.syndication.views import Feed
+    from django.urls import reverse
+
+
+    class FeedWithStylesheetView(Feed):
+        ...  # author, etc.
+        xsl_stylesheet_url = reverse("your-custom-view-name")
```

Comment:
🤔  Does this example need to use `reverse_lazy` by convention, or is continuing to use `reverse`[^1] going to be OK here?

[^1]: As earlier in the document, but at method evaluation level, not at module level.